portal/sentry.client.config.ts

```diff
@@ -62,6 +62,7 @@ function enableSentry() {
     denyUrls: [
       process.env.NEXT_PUBLIC_POINTS_URL,
       process.env.NEXT_PUBLIC_TOKEN_PRICES_URL,
+      process.env.NEXT_PUBLIC_TVL_URL,
       process.env.NEXT_PUBLIC_COOKIE3_URL,
       // filter in case any is undefined, although in prod all should be defined.
```

Comment:
I know they were not sorted already (and it was me who did not do it 😓 ), but can we get these sorted? 😅 

```suggestion
      process.env.NEXT_PUBLIC_COOKIE3_URL,
      process.env.NEXT_PUBLIC_POINTS_URL,
      process.env.NEXT_PUBLIC_TOKEN_PRICES_URL,
      process.env.NEXT_PUBLIC_TVL_URL,
      // filter in case any is undefined, although in prod all should be defined.
```